### ISSUE TITLE
OOM killed syncing schlesi

### ISSUE THREAD

**Issue by wemeetagain** [ChainSafe/lodestar#920]
**Describe the bug**
Syncing schlesi, got an OOM crash.

**Steps to Reproduce**
<!--Steps to reproduce the behavior:
-->

**Screenshots**
error logs: note, the two lines ` info: Fetching blocks for 20919...20983 slot range` `info: Imported blocks for slots:` were repeated as far back as my console history showed
```
2020-05-15 11:51:08  []                 info: Fetching blocks for 20919...20983 slot range
2020-05-15 11:51:08  []                 info: Imported blocks for slots:                                                                                                                 
2020-05-15 11:51:08  []                 info: Fetching blocks for 20919...20983 slot range
2020-05-15 11:51:08  []                 info: Imported blocks for slots:               
2020-05-15 11:51:08  []                 info: Fetching blocks for 20919...20983 slot range                                                                                               
                                                                                                                                                                                         
<--- Last few GCs --->                                                                      
                                                                                            
[1194148:0x3b360a0]  3821808 ms: Scavenge 2031.5 (2059.1) -> 2023.2 (2059.6) MB, 8.9 / 0.0 ms  (average mu = 0.182, current mu = 0.118) allocation failure 
[1194148:0x3b360a0]  3821849 ms: Scavenge 2032.0 (2059.6) -> 2023.9 (2060.1) MB, 9.5 / 0.0 ms  (average mu = 0.182, current mu = 0.118) allocation failure 
[1194148:0x3b360a0]  3823606 ms: Mark-sweep 2032.7 (2060.1) -> 2023.9 (2073.1) MB, 1725.8 / 0.0 ms  (average mu = 0.130, current mu = 0.080) allocation failure scavenge might not succee
d                                                                                           


<--- JS stacktrace --->

==== JS stack trace =========================================

    0: ExitFrame [pc: 0x1374fd9]
Security context: 0x16c11c3408a1 <JSObject>
    1: _write [0x82bc08b4f71] [/home/cayman/Code/lodestar/node_modules/winston-transport/index.js:~68] [pc=0x142897e65dee](this=0x0d382e7702e9 <TransportStream map = 0x1a0a03bdf921>,0x2
966115ee279 <Object map = 0x2d32541a7581>,0x126f3889e1b1 <String[#4]: utf8>,0x018539a5b369 <JSFunction WritableState.onwrite (sfi = 0x270eab58c9f1)>)
    2: ondata [0xd382e770679] ...

FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory
 1: 0x9da7c0 node::Abort() [node]
 2: 0x9db976 node::OnFatalError(char const*, char const*) [node]
 3: 0xb39f1e v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [node]
 4: 0xb3a299 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [node]
 5: 0xce5635  [node]
 6: 0xce5cc6 v8::internal::Heap::RecomputeLimits(v8::internal::GarbageCollector) [node]
 7: 0xcf1b5a v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [node]
 8: 0xcf2a65 v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [node]
 9: 0xcf410f v8::internal::Heap::HandleGCRequest() [node]
10: 0xca30f4 v8::internal::StackGuard::HandleInterrupts() [node]
11: 0xff1af7 v8::internal::Runtime_StackGuard(int, unsigned long*, v8::internal::Isolate*) [node]
12: 0x1374fd9  [node]
Aborted (core dumped)

```
some graphs:
[lodecli-0515.pdf](https://github.com/ChainSafe/lodestar/files/4636044/lodecli-0515.pdf)

**Comment by mpetrunic** [comment]
Think this is resolved using #916

**Comment by wemeetagain** [comment]
seemingly resolved
